README.md

```diff
@@ -1 +1,136 @@
-# Hello
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-microverse] <a name="Hello-microverse"></a>
+
+**[Hello-microverse]** this was mini-projects that upskill to create a feature branch and test linters in order to have professional-looking code.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
+
+  <summary>Version control</summary>
+  <ul>
+    <li><a href="github.com">Git Hub</a></li>
+  </ul>
+</details
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Creating your first "Hello-microverse" project
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A Git hub account
+<br>
+-Node JS
+<br>
+-Visual Studio Code or any other code editor of your preference
+<br>
+-web browser
+
+### Setup
+
+Clone this repository to your desired folder:<br>
+git@github.com:brighton0102/Hello.git
+
+## Install Linters
+you can use 
+
+Github
+VS code
+so that you can access the code
+
+To run the project, execute the following commands
+
+Linters
+git commands
+
+### Run test
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: (https://github.com/brighton0102)
+- Twitter: (@brighton_ui)
+- LinkedIn: (in/brighton-mtandi-976615267)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions and feature request are welcome.
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```

Comment:
- [ ] As your previous code reviewer humbly requested you to remove `placeholders` but I can see you haven't removed them. Please remove the placeholder because it is meant for developers while create `documentation` professionally. Leaving these kinds of `placeholders` makes your `README.md` file unprofessional.